frontend/src/containers/Collective.js

```diff
@@ -0,0 +1,273 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
The problem with adding new files, is that it's impossible for me to see what you changed in them.
E.g. How different is this page compared to the former PublicGroupPage.js?